setup.py

```diff
@@ -27,6 +27,7 @@
 
 Source code: https://github.com/pvlib/pvlib-python
 """
+LONG_DESCRIPTION_CONTENT_TYPE = "text/plain"
```

Comment:
```suggestion
LONG_DESCRIPTION_CONTENT_TYPE = "text/x-rst"
```

Although we don't currently need any RST features in this description, I think we should stick with `text/x-rst` as the type because we know it works (since twine has been defaulting to it so far).  I see some `text/plain` projects end up rendering like this: https://pypi.org/project/plaso/